packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -502,6 +505,30 @@ class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>>
     };
     _updateActions();
     _updateOverlay();
+
+    WidgetsBinding.instance.addObserver(this);
+    WidgetsBinding.instance.addPostFrameCallback((_) => _getFieldWidth());
+  }
+
+  bool _isOffstage = true;
+  double? _fieldWidth;
+  final GlobalKey _textFieldKey = GlobalKey();
```

Comment:
Nit: Maybe put these instance variables at the top of the class with the others.